src/pyspextool/telluric/telluric_correction.py

```diff
@@ -141,6 +158,16 @@ def telluric_correction(object_file, standard, standard_file, output_name,
     if verbose is None:
         verbose = setup.state['verbose']
 
+    if verbose is True:
+        logging.getLogger().setLevel(logging.INFO)
+        setup.state["verbose"] = True
+        
+    elif verbose is False:
+        logging.getLogger().setLevel(logging.ERROR)
+        setup.state["verbose"] = False
+
+    logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
+    logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.DEBUG)            
```

Comment:
You shouldn't need these logging config statements. We should set this up somewhere else which is called by everything else.  At least put one of these lines outside of the function.

Reply:
Done.